src/widgets/WordListWidget.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="word-list-widget u-widget u-flex">
+    <WordList v-if="filteredWordList.length > 0" :wordList="filteredWordList" />
+  </div>
+</template>
+
+<script>
+  import WIDGETS_NS, { WordList } from '@scaife-viewer/scaife-widgets';
+  import { MODULE_NS } from '@/reader/constants';
+
+  export default {
+    name: 'WordListWidget',
+    scaifeConfig: {
+      displayName: 'Word List',
+    },
+    components: {
+      WordList,
+    },
+    created() {
+      if (this.enabled) {
+        this.fetchData();
+      }
+    },
+    data() {
+      return {
+        wordList: [],
+      };
+    },
+    watch: {
+      passage: 'fetchData',
+    },
+    computed: {
+      selectedToken() {
+        return this.$store.state[MODULE_NS].selectedToken;
+      },
+      filteredWordList() {
+        if (this.selectedToken === null) {
+          return this.wordList;
+        }
+        return this.wordList.filter(w => w.text === this.selectedToken.value);
```

Comment:
@paltman this filter is against the token value, but the filter in scaife-viewer/scaife-viewer is against the lemma:

https://github.com/scaife-viewer/scaife-viewer/blob/227a933c2b803bdf9d247f2d201219f07301e209/static/src/js/reader/widgets/WidgetWordList.vue#L8

https://scaife.perseus.org/reader/urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.1-1.7/?highlight=%40Ἀχιλῆος%5B1%5D

<img width="680" alt="image" src="https://user-images.githubusercontent.com/629062/84851230-52ed9c80-b01f-11ea-8a9a-0bdf1d1860bc.png">

https://deploy-preview-30--explorehomer-dev.netlify.app/reader?urn=urn%3Acts%3AgreekLit%3Atlg0012.tlg001.perseus-grc2%3A1.1-1.7

<img width="680" alt="image" src="https://user-images.githubusercontent.com/629062/84851239-5b45d780-b01f-11ea-9e03-0245fc53135f.png">